### ISSUE TITLE
Cannot start Navigator on OSX

### ISSUE THREAD

**Issue by wavetossed** [ContinuumIO/anaconda-issues#1148]
When the GUI app would not start, I poked around in the .app folder and discovered that it runs anaconda-navigator. So I tried that from the shell prompt and get these errors:

Michaels-MacBook-Pro:~ mpd$ anaconda-navigator
Traceback (most recent call last):
  File "/Users/mpd/anaconda/lib/python3.5/site-packages/qtpy/__init__.py", line 119, in <module>
    from PySide import __version__ as PYSIDE_VERSION  # analysis:ignore
ImportError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/mpd/anaconda/bin/anaconda-navigator", line 4, in <module>
    import anaconda_navigator.app.main
  File "/Users/mpd/anaconda/lib/python3.5/site-packages/anaconda_navigator/app/main.py", line 16, in <module>
    from qtpy.QtCore import QCoreApplication, QEvent, QObject, Qt
  File "/Users/mpd/anaconda/lib/python3.5/site-packages/qtpy/__init__.py", line 125, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found
 
conda install pyside refuses to work. What now?

**Comment by goanpeca** [comment]
@wavetossed could you type in the command prompt

`conda info`

and `conda list`

and paste the output here?

**Comment by wavetossed** [comment]
I ended up completely removing Anaconda and some other python installs. Then I  tried again with Anaconda 3 which worked to a point. Navigator was mostly OK but Jupiter ran into problems. Deleted again and started with Anaconda 2. Then created python 3.5 environments  to run Jupyter, another R environment also to run Jupyter and all the problems are gone.

At this point I think the best way to use Python 3.5 is to install Anaconda 2 and use the Navigator to create Environments for Python 3.5 projects. In the process I came to understand the features of Navigator and Jupyter better.

The Pyside issue was related to not having any Python 2.7 under the hood. Installing Anaconda 2 fixes that
